src/js/themes/base.js

```diff
@@ -305,13 +305,28 @@ export const generate = (baseSpacing = 24, scale = 6) => { // 24
         radius: '4px',
         width: '2px',
       },
+      box: {
+        // extend: undefined,
+      },
       check: {
         // color: { dark: undefined, light: undefined },
         width: '4px',
       },
+      container: {
+        // extend: undefined,
+      },
+      icon: {
+        // extend: undefined,
+      },
       icons: {
         // checked: undefined,
       },
+      input: {
+        // extend: undefined,
+      },
+      knob: {
```

Comment:
How is this different than `check`?

Reply:
This is the override for the toggle only. we have the toggle override and the regular checkbox override.

Reply:
I think we can leverage the existing `toggle` prop

Reply:
nahh.. we need both :(